test/lib/usd/translators/CMakeLists.txt

```diff
@@ -112,3 +112,23 @@ mayaUsd_add_test(testUsdImportUVSetsFloat
         "PIXMAYA_READ_FLOAT2_AS_UV=1"
 )
 set_property(TEST testUsdImportUVSetsFloat APPEND PROPERTY LABELS translators)
+
+mayaUsd_add_test(testUsdExportCustomConverter
+    PYTHON_MODULE testUsdExportCustomConverter
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+    ENV
+        "MAYA_PLUG_IN_PATH=${CMAKE_CURRENT_BINARY_DIR}/../plugin"
+        "PXR_PLUGINPATH_NAME=${CMAKE_CURRENT_BINARY_DIR}/../plugin"
+)
+set_property(TEST testUsdExportCustomConverter APPEND PROPERTY LABELS translators)
+
+# Test uses standardSurface, which was introduced in Maya 2020.
+if (MAYA_APP_VERSION VERSION_GREATER_EQUAL 2020) 
```

Comment:
After reading through the tests, it's looking like maybe the 2020+ check was intended to guard the `testUsdExportCustomConverter` test instead?